### ISSUE TITLE
feature request: Add option to set custom space wallpapers

### ISSUE THREAD

**Issue by eliabexp** [anyproto/anytype-kotlin#1026]
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing requests and didn't find any that were similar
- [X] I have considered creating a pull request instead and want to proceed

### Clear and concise description of the problem

There are color options to use as the space wallpaper, it would be cool if we could add images as the space wallpaper

### Suggested solution

Add an option to use a custom wallpaper in space personalization settings

**Comment by candroid-man** [comment]
Yeah the blue gradient thing is not doing it for me, I'd rather it be just a plain black wallpaper

**Comment by ProfessorDey** [comment]
As someone who is far and away the most introspective at night/in bed, I too badly want an alternative to the universally blindingly bright space wallpapers on the app. Just a pure black wallpaper would be enough even. Being able to load my own would be even better, or if nothing else, why not let ut be an arbitrary css hex value?
